EventFilter/HcalRawToDigi/src/HcalUnpacker.cc

```diff
@@ -404,7 +405,7 @@ void HcalUnpacker::unpackVME(const FEDRawData& raw, const HcalElectronicsMap& em
 	  ncurr=0;
 	  valid=true;
 	}      
-	// add the word (if within settings or recent firmware [recent firmware ignores startSample/endSample])
+	// add the word (if within settings or recent firmware [recent firmware Suppressing startSample/endSample])
```

Comment:
"Suppressing"? (maybe inadvertently replaced by a copy/paste?)

Reply:
must have been, I'll revert it